docs/workforce-management-framework/topics/ref-keycloak-securitystrategy.adoc

```diff
@@ -1,38 +1,38 @@
-. xref:defining-keycloak-access-control-{chapter}[Defining {WFM-RC-KeycloakName} Access Control]
-. xref:defining-keycloak-password-storage-{chapter}[Defining {WFM-RC-KeycloakName} Password Storage]
-. xref:keycloak-authentication-{chapter}[{WFM-RC-KeycloakName} Authentication]
-. xref:keycloak-authorization-{chapter}[{WFM-RC-KeycloakName} Authorization]
+. xref:{context}-understanding-keycloak-session-management[Understanding {WFM-RC-KeycloakName} Session Management]
+. xref:{context}-defining-keycloak-access-control[Defining {WFM-RC-KeycloakName} Access Control]
+. xref:{context}-defining-keycloak-password-storage[Defining {WFM-RC-KeycloakName} Password Storage]
+. xref:{context}-keycloak-authentication[{WFM-RC-KeycloakName} Authentication]
+. xref:{context}-keycloak-authorization[{WFM-RC-KeycloakName} Authorization]
 
-[id='understanding-keycloak-session-management-{chapter}']
+[id='{context}-understanding-keycloak-session-management']
 == Understanding {WFM-RC-KeycloakName} Session Management
 The _express-session_ module provides session management functionality and
 the management of sessions is performed on the link:{WFM-RC-KeycloakURL}server_admin/topics/sessions/administering.html[*{WFM-RC-KeycloakName}* server]
 For more information on _express-session_, see link:https://github.com/expressjs/session[express-session].
 
-[id='defining-keycloak-access-control-{chapter}']
+[id='{context}-defining-keycloak-access-control']
 == Defining {WFM-RC-KeycloakName} Access Control
 *{WFM-RC-KeycloakName}* provides role based access control and is configured on the *{WFM-RC-KeycloakName}* server.
 For more information about role based access control, see link:{WFM-RC-KeycloakURL}server_admin/topics/roles.html[*{WFM-RC-KeycloakName}* Roles].
 
-To see an example of how roles are protected, see the *{WFM-RC-KeycloakName}* link:./pro-Keycloak-implementaion.adoc[documentation].
+For more information on how roles are protected, see the *{WFM-RC-KeycloakName}* link:./pro-Keycloak-implementaion.adoc[documentation].
 
-[id='defining-keycloak-password-storage-{chapter}']
+[id='{context}-defining-keycloak-password-storage']
 == Defining {WFM-RC-KeycloakName} Password Storage
 *{WFM-RC-KeycloakName}* uses the link:https://en.wikipedia.org/wiki/PBKDF2[BKDF2 algorithm] for password hashing by default.
 *{WFM-RC-KeycloakName}* also provides enforced link:{WFM-RC-KeycloakURL}server_admin/topics/authentication/password-policies.html[password policies] from the server.
 
-For further reading, see link:{WFM-RC-KeycloakURL}server_admin/topics/identity-broker.html[identity-broker.html] and link:{WFM-RC-KeycloakURL}server_admin/topics/user-federation.html[user federation.html].
+For more information on server administration topics, see link:{WFM-RC-KeycloakURL}server_admin/topics/identity-broker.html[identity-broker] and link:{WFM-RC-KeycloakURL}server_admin/topics/user-federation.html[user federation].
```

Comment:
I added "server administration" as this is the section the links are found under in the keycloak docs